### ISSUE TITLE
Command TodoTrouble not working

### ISSUE THREAD

**Issue by Chaitanyabsprip** [folke/todo-comments.nvim#26]
command :TodoTrouble causes nvim to freeze, I can't do anything and the only way I could find out to get out of it is to close the terminal. 
```lua

use {
  "folke/todo-comments.nvim",
  config = function()
    require("todo-comments").setup {}
  end
}
u.keymap("n", "<leader>gt", "TodoTelescope", ntst)
```
This is my config for todo-comments.
I am using latest nvim commit. 
and I am on arch linux.

**Comment by folke** [comment]
Does `TodoTelescope` work? How many results? Is your working directory correct?

**Comment by Chaitanyabsprip** [comment]
:TodoTelescope works fine and gives a lot of results. I invoked the command in my nvim directory and I've kept my lsp servers there, so there are tonnes of TODOs. also yes, the directory is right.

**Comment by folke** [comment]
Did it used to work? Or did you never try before?

**Comment by folke** [comment]
I think I found what was causing this. I actually made a change to trouble yesterday that incorporated some of my changes to Neovim core, to increase performance of lsp related trouble lists up to 400x :) (when using the latest Neovim 0.50)

But by doing so, I made a small typo that actually increased the time it takes to get the todos, especially on neovim versions older than two days.

Either way, it should be fixed now. Thank you for reporting!
